apps/issuer/api.py

```diff
@@ -143,6 +196,41 @@ def post(self, request, **kwargs):
         return super(AllBadgeClassesList, self).post(request, **kwargs)
 
 
+class AllCollectionBadgeClassesList(UncachedPaginatedViewMixin, BaseEntityListView):
+    """
+    GET a list of collectionbadgeclasses within one issuer context or
+    POST to create a new collectionbadgeclass within the issuer context
+    """
+    model = CollectionBadgeContainer
+    permission_classes = [
+        IsServerAdmin
```

Comment:
Why does one have to be server admin?

Reply:
You`re right, i extended the permission classes